docs/functions.rst

```diff
@@ -1572,6 +1572,44 @@ Example:
         args:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
+.. _deletedatausingkopiaserver:
+DeleteDataUsingKopiaServer
+--------------------------
+
+This function deletes the snapshot data backed up by the ``BackupDataUsingKopiaServer``
+function. It creates a new Pod that runs ``delete data`` job.
```

Comment:
is `delete data` a command? If not, I don't think it should be in ``. 
And if it's a command should we use job instead of command?

Reply:
I don't think Kanister controller creates a job. It just creates a pod dynamically. If we run `delete snapshot` command in that pod, I think we should mention
```
It creates a new pod that runs ``delete snapshot`` command.
```
WDYT?

Reply:
Got it ! delete snapshot looks good

Reply:
will make the changes